packages/jaeger-ui/package.json

```diff
@@ -80,7 +79,7 @@
     "react-redux": "^8.1.2",
     "react-router-dom": "5.3.4",
     "react-router-dom-v5-compat": "^6.24.0",
-    "react-vis": "^1.7.2",
+    "react-vis": "1.11.12",
```

Comment:
1.11.12 is the current version in Jaeger UI, and the latest has some issues (see https://github.com/uber/react-vis/issues/1485#issuecomment-1913666895)